openwisp_monitoring/device/templates/admin/config/device/change_form.html

```diff
@@ -141,6 +141,31 @@ <h2>{% trans 'Interface status' %}: {{ interface.name }}</h2>
                   {% endif %}
                   {% endif %}
               </div>
+              <div class = "device_addresses">
```

Comment:
the css class should use the dash for consistency.
The div is not needed here, you can elminate the div and move the class to the table element